lib/interfaces/src/DrivebrainETHInterface.cpp

```diff
@@ -6,30 +6,29 @@ hytech_msgs_MCUOutputData DrivebrainETHInterface::make_db_msg(const SharedCarSta
 {
     hytech_msgs_MCUOutputData out;
     out.accel_percent = shared_state.pedals_data.accelPercent;
-    out.brake_percent = shared_state.pedals_data.brakePercent;
-    
+    out.brake_percent = shared_state.pedals_data.regenPercent;
+
```

Comment:
Is this purely a refactoring or does this actually indicate how much we are regenning?